src/content/changes/highlights/2022-09-30.md

```diff
@@ -0,0 +1,20 @@
+---
+date: 2022-09-30T10:00:00
+title: Highlights for the week ending September 30, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## User interfaces
+
+The `kubectl gs login` command as of [v2.23.0](https://github.com/giantswarm/kubectl-gs/releases/tag/v2.23.0) provides a flag `--cn-prefix`. This allows to specify the CN of a client certificate issued for workload cluster access, so that the client using the certificate can be mapped to RBAC rules which are bound to a user.
+
+## Apps
+- [dex-app](https://github.com/giantswarm/dex-app) now provides Dex [v2.34.0](https://github.com/dexidp/dex/releases/tag/v2.34.0).
+- [fluent-logshipping-app](https://github.com/giantswarm/fluent-logshipping-app) version [v2.0.0](https://github.com/giantswarm/fluent-logshipping-app/blob/master/CHANGELOG.md#200---2022-09-28): this update supports CRI instead of the docker Format and changes the storage path to not use a tmpfs to not overload the nodes.
+
```

Comment:
```suggestion
## Apps

- [dex-app](https://github.com/giantswarm/dex-app) now provides Dex [v2.34.0](https://github.com/dexidp/dex/releases/tag/v2.34.0).
- [fluent-logshipping-app](https://github.com/giantswarm/fluent-logshipping-app) version [v2.0.0](https://github.com/giantswarm/fluent-logshipping-app/blob/master/CHANGELOG.md#200---2022-09-28): this update supports CRI instead of the docker Format and changes the storage path to not use a tmpfs to not overload the nodes.
- [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.11.2](https://github.com/giantswarm/kyverno-app/blob/master/CHANGELOG.md#0112---2022-09-26) updates to upstream version 1.7.3.
- [security-pack]https://github.com/giantswarm/security-pack) version [0.8.0](https://github.com/giantswarm/security-pack/blob/main/CHANGELOG.md#080---2022-09-27) includes new versions of all pack components (listed in this announcement), and finishes the switch to Trivy Operator as the default vulnerability scan manager. It is also the first release to be included in the `giantswarm` catalog. It will be removed from the `playground` catalog in a future release.
- [security-pack-helper](https://github.com/giantswarm/security-pack-helper) version [v0.0.2](https://github.com/giantswarm/security-pack-helper/blob/main/CHANGELOG.md#added) initial release supports watching and deleting Kyverno `ReportChangeRequest` resources when they exceed a configured threshold.
- [starboard-exporter](https://github.com/giantswarm/starboard-exporter) version [v0.6.0](https://github.com/giantswarm/starboard-exporter/blob/main/CHANGELOG.md#060---2022-09-16) disables reconciliation of `CISKubeBenchReports` by default in support of the switch to Trivy Operator.
- [trivy-app](https://github.com/giantswarm/trivy-app) version [v0.6.0](https://github.com/giantswarm/trivy-app/blob/main/CHANGELOG.md#060---2022-09-16) updates to Trivy version `0.30.4`.
- [trivy-operator-app](https://github.com/giantswarm/trivy-operator-app) version [v0.2.0](https://github.com/giantswarm/trivy-operator-app/blob/main/CHANGELOG.md#020---2022-09-26) updates to Trivy Operator version `0.2.1`, adding support for calling custom webhooks and exposing more configuration options for scan jobs.

```